js/private/BUILD.bazel

```diff
@@ -24,9 +24,9 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
-config_setting(
+alias(
     name = "experimental_allow_unresolved_symlinks",
-    values = {"experimental_allow_unresolved_symlinks": "true"},
+    actual = "@aspect_config_settings//:allow_unresolved_symlinks",
```

Comment:
Actually it sounds like this belongs in https://github.com/bazel-contrib/bazel_features - the whole point there is to make it possible to write rules that detect bazel features are present/not present. Then we'd only have to do these tricks in one place and have other Bazel experts like @fmeum who can help review.

Reply:
problem with bazel_features is that it requires two macro calls in the WORKSPACE, which we don't have in rules_js. so we can only call repository rules that belong to rules_js.

Reply:
one http_archive and then features_deps call to load deps. but in rules_js we have register_node_toolchains and rules_js_dependencies.